examples/automm/object_detection/detection_train.py

```diff
@@ -97,4 +109,7 @@ def detection_train(
         lr=args.lr,
         wd=args.wd,
         epochs=args.epochs,
+        num_gpus=args.num_gpus,
+        val_metric=args.val_metric,  # "mAP" or "direct_loss" or None (use default: "direct_loss")
```

Comment:
I previously commented about using `mAP` instead of `map`. After a second thought, I think may be `map` is easier to remember.

Reply:
Sure, changing back to map, but keeping both `val_map` and `val_mAP` output keys in case users use `mAP` to retrieve results in their script.